src/sentry/static/sentry/app/routes.jsx

```diff
@@ -1151,40 +1153,119 @@ function routes() {
           <IndexRoute component={errorHandler(ProjectInstallOverview)} />
           <Route path=":platform/" component={errorHandler(ProjectInstallPlatform)} />
         </Route>
-
-        <Redirect from=":projectId/" to="/organizations/:orgId/issues/" />
-        <Redirect from=":projectId/issues/" to="/organizations/:orgId/issues/" />
-        <Redirect
-          from=":projectId/searches/:searchId/"
-          to="/organizations/:orgId/issues/"
-        />
-        <Redirect from=":projectId/dashboard/" to="/organizations/:orgId/dashboards/" />
-        <Redirect from=":projectId/releases/" to="/organizations/:orgId/releases/" />
-        <Redirect
-          from=":projectId/releases/:version/"
-          to="/organizations/:orgId/releases/:version/"
-        />
-        <Redirect
-          from=":projectId/releases/:version/new-events/"
-          to="/organizations/:orgId/releases/:version/new-events/"
-        />
-        <Redirect
-          from=":projectId/releases/:version/all-events/"
-          to="/organizations/:orgId/releases/:version/all-events/"
-        />
-        <Redirect
-          from=":projectId/releases/:version/artifacts/"
-          to="/organizations/:orgId/releases/:version/artifacts/"
-        />
-        <Redirect
-          from=":projectId/releases/:version/commits/"
-          to="/organizations/:orgId/releases/:version/commits/"
-        />
-
-        <Redirect
-          from=":projectId/user-feedback/"
-          to="/organizations/:orgId/user-feedback/"
-        />
+        <Route path=":projectSlug/">
```

Comment:
Can we keep it `projectId` to be consistent with the rest of the routes (even though it's confusing).